lib/transformers/add_state.ex

```diff
@@ -5,9 +5,11 @@ defmodule AshStateMachine.Transformers.AddState do
   alias Spark.Dsl.Transformer
 
   def before?(Ash.Resource.Transformers.DefaultAccept), do: true
+  def before?(AshStateMachine.Transformers.SetDefaultInitialState), do: false
   def before?(_), do: false
 
   def after?(Ash.Resource.Transformers.DefaultAccept), do: false
+  def after?(AshStateMachine.Transformers.SetDefaultInitialState), do: true
```

Comment:
Weird that we have this twice in the current stuff. Only need to define the one that returns true as the default is false